kenji_crosland/app/sass/colors/_theme.scss

```diff
@@ -0,0 +1,19 @@
+//Color Scheme
+//https://color.adobe.com/pomegranate-explosion-color-theme-7287945/
+//'Pomegranate Explosion' Color Palate
+$dark-turquoise: #63A69F;
+$linen: #FBF5E5;
+$wheat: #F2E1AC;
+$salmon: #F2836B;
+$carnation: #F2594B;
+$cardinal-red: #CD2C24;
+$light-gray: #eef;
+
+//Theme variables
+$light-shading: $light-gray;
+$background: $wheat;
+$light-background: $linen;
+$primary1: $salmon;
+$primary2: $dark-turquoise;
+$accent: $carnation;
+$strong-accent: $cardinal-red;
```

Comment:
Very nice abstraction of your color-named variables!  I was all ready to get mean about naming variables after colors, but this is very well done.  :)